expr/literals.go

```diff
@@ -404,6 +404,10 @@ type ProtoLiteral struct {
 func (*ProtoLiteral) isRootRef()            {}
 func (t *ProtoLiteral) GetType() types.Type { return t.Type }
 func (t *ProtoLiteral) String() string {
+	switch literalType := t.Type.(type) {
+	case types.PrecisionTimeStampType, types.PrecisionTimeStampTzType:
+		return fmt.Sprintf("%s(%d)", literalType, t.Value.(uint64))
```

Comment:
it's likely not necessary to do the type assertion here with Sprintf which should properly handle the interface type anyways.

Reply:
fixed it